src/AudioPlayer.ts

```diff
@@ -3,6 +3,12 @@ import EventTarget from './EventTarget';
 
 import ChromeAudioContext, { ChromeHTMLAudioElement, MediaStreamAudioDestinationNode } from './ChromeAudioContext';
 
+/**
+ * HTMLMediaElement srcObject
+ * @private
+ */
+export type ISourceObject = any;
```

Comment:
Can we use the actual types?
https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/srcObject

`MediaStream | MediaSource | Blob | undefined`

Reply:
Cool, done